configure.ac

```diff
@@ -24,6 +24,9 @@ AC_CONFIG_HEADERS([config.h])
 dnl Where are the sources
 AC_CONFIG_SRCDIR([src/main.c])
 
+dnl dependencies
+AC_CHECK_LIB(crypto, SHA1_Init)
```

Comment:
Just checking: you've checked that `libcrypro` is enough and no need for `libssl`, right?

Reply:
Yes, i tried compiling and running and it's enough. You can try yourself.

Although i noticed that i get a wall of deprecated function warnings regarding the `libcrypto` functions, apparently they are deprecated starting with `Openssl 3.0`+. Do we not care?

Reply:
My code was definitely for OpenSSL 1.x.

I suppose an update is needed at some point. I opened #32 for this.